diff --git a/files/mygui/openmw_settings_window.layout b/files/mygui/openmw_settings_window.layout
index 788672c..dbc0905 100644
--- a/files/mygui/openmw_settings_window.layout
+++ b/files/mygui/openmw_settings_window.layout
@@ -359,9 +359,12 @@
           <Property key="Caption" value="#{arenamp=settings.subsection.display}"/>
           <Widget type="ScrollView" skin="MW_ScrollView" position="4 4 478 400" align="Left Top" name="DisplayScroll">
             <Property key="CanvasAlign" value="Left Top"/>
-            <Property key="CanvasSize" value="458 630"/>
-            <Widget type="ListBox" skin="MW_List" position="8 8 220 142" name="ResolutionList"/>
+            <Property key="CanvasSize" value="458 360"/>
+            <Widget type="ListBox" skin="MW_List" position="8 8 220 142" name="ResolutionList">
+              <Property key="Visible" value="false"/>
+            </Widget>
             <Widget type="HBox" skin="" position="244 8 206 24">
+              <Property key="Visible" value="false"/>
               <Widget type="AutoSizedButton" skin="MW_Button" position="0 0 24 24" name="FullscreenButton">
                 <UserString key="SettingCategory" value="Video"/>
                 <UserString key="SettingName" value="fullscreen"/>
@@ -371,7 +374,7 @@
                 <Property key="Caption" value="#{arenamp=settings.fullscreen}"/>
               </Widget>
             </Widget>
-            <Widget type="HBox" skin="" position="244 38 206 24">
+            <Widget type="HBox" skin="" position="8 8 300 24">
               <Widget type="AutoSizedButton" skin="MW_Button" position="0 0 24 24" name="VSyncButton">
                 <UserString key="SettingCategory" value="Video"/>
                 <UserString key="SettingName" value="vsync"/>
@@ -382,6 +385,7 @@
               </Widget>
             </Widget>
             <Widget type="HBox" skin="" position="244 68 206 24">
+              <Property key="Visible" value="false"/>
               <Widget type="AutoSizedButton" skin="MW_Button" position="0 0 24 24" name="WindowBorderButton">
                 <UserString key="SettingCategory" value="Video"/>
                 <UserString key="SettingName" value="window border"/>
@@ -391,10 +395,10 @@
                 <Property key="Caption" value="#{arenamp=settings.window_border}"/>
               </Widget>
             </Widget>
-            <Widget type="TextBox" skin="NormalText" position="8 236 442 18" name="FovText">
+            <Widget type="TextBox" skin="NormalText" position="8 44 442 18" name="FovText">
               <Property key="Caption" value="#{arenamp=settings.fov}"/>
             </Widget>
-            <Widget type="ScrollBar" skin="MW_HScroll" position="8 258 442 16">
+            <Widget type="ScrollBar" skin="MW_HScroll" position="8 66 442 16">
               <Property key="Range" value="81"/>
               <Property key="Page" value="1"/>
               <UserString key="SettingType" value="Slider"/>
@@ -406,17 +410,19 @@
               <UserString key="SettingLabelWidget" value="FovText"/>
               <UserString key="SettingLabelCaption" value="#{arenamp=settings.fov}"/>
             </Widget>
-            <Widget type="TextBox" skin="SandText" position="8 276 80 16">
+            <Widget type="TextBox" skin="SandText" position="8 84 80 16">
               <Property key="Caption" value="#{sLow}"/>
             </Widget>
-            <Widget type="TextBox" skin="SandText" position="370 276 80 16">
+            <Widget type="TextBox" skin="SandText" position="370 84 80 16">
               <Property key="Caption" value="#{sHigh}"/>
               <Property key="TextAlign" value="Right"/>
             </Widget>
-            <Widget type="TextBox" skin="NormalText" position="8 302 442 18" name="GammaText">
+            <Widget type="TextBox" skin="NormalText" position="8 104 442 18" name="GammaText">
+              <Property key="Visible" value="false"/>
               <Property key="Caption" value="#{sGamma_Correction}"/>
             </Widget>
-            <Widget type="ScrollBar" skin="MW_HScroll" position="8 324 442 16" name="GammaSlider">
+            <Widget type="ScrollBar" skin="MW_HScroll" position="8 126 442 16" name="GammaSlider">
+              <Property key="Visible" value="false"/>
               <Property key="Range" value="10000"/>
               <Property key="Page" value="300"/>
               <UserString key="SettingType" value="Slider"/>
@@ -426,14 +432,16 @@
               <UserString key="SettingMin" value="0.1"/>
               <UserString key="SettingMax" value="3.0"/>
             </Widget>
-            <Widget type="TextBox" skin="SandText" position="8 342 100 16" name="GammaTextDark">
+            <Widget type="TextBox" skin="SandText" position="8 144 100 16" name="GammaTextDark">
+              <Property key="Visible" value="false"/>
               <Property key="Caption" value="#{sDark_Gamma}"/>
             </Widget>
-            <Widget type="TextBox" skin="SandText" position="350 342 100 16" name="GammaTextLight">
+            <Widget type="TextBox" skin="SandText" position="350 144 100 16" name="GammaTextLight">
+              <Property key="Visible" value="false"/>
               <Property key="Caption" value="#{sLight_Gamma}"/>
               <Property key="TextAlign" value="Right"/>
             </Widget>
-            <Widget type="Widget" skin="" position="8 368 442 42" name="AnisotropyBox">
+            <Widget type="Widget" skin="" position="8 110 442 42" name="AnisotropyBox">
               <Widget type="TextBox" skin="NormalText" position="0 0 442 18" name="AnisotropyLabel">
                 <Property key="Caption" value="#{arenamp=settings.anisotropy}"/>
               </Widget>
@@ -450,10 +458,10 @@
                 <UserString key="SettingLabelCaption" value="#{arenamp=settings.anisotropy}"/>
               </Widget>
             </Widget>
-            <Widget type="TextBox" skin="NormalText" position="8 554 442 18">
+            <Widget type="TextBox" skin="NormalText" position="8 296 442 18">
               <Property key="Caption" value="#{sTransparency_Menu}"/>
             </Widget>
-            <Widget type="ScrollBar" skin="MW_HScroll" position="8 576 442 16">
+            <Widget type="ScrollBar" skin="MW_HScroll" position="8 318 442 16">
               <Property key="Range" value="10000"/>
               <Property key="Page" value="300"/>
               <UserString key="SettingType" value="Slider"/>
@@ -461,17 +469,17 @@
               <UserString key="SettingName" value="menu transparency"/>
               <UserString key="SettingValueType" value="Float"/>
             </Widget>
-            <Widget type="TextBox" skin="SandText" position="8 594 80 16">
+            <Widget type="TextBox" skin="SandText" position="8 336 80 16">
               <Property key="Caption" value="#{sFull}"/>
             </Widget>
-            <Widget type="TextBox" skin="SandText" position="370 594 80 16">
+            <Widget type="TextBox" skin="SandText" position="370 336 80 16">
               <Property key="Caption" value="#{sNone}"/>
               <Property key="TextAlign" value="Right"/>
             </Widget>
-            <Widget type="TextBox" skin="NormalText" position="8 488 442 18">
+            <Widget type="TextBox" skin="NormalText" position="8 230 442 18">
               <Property key="Caption" value="#{sMenu_Help_Delay}"/>
             </Widget>
-            <Widget type="ScrollBar" skin="MW_HScroll" position="8 510 442 16">
+            <Widget type="ScrollBar" skin="MW_HScroll" position="8 252 442 16">
               <Property key="Range" value="10000"/>
               <Property key="Page" value="300"/>
               <UserString key="SettingType" value="Slider"/>
@@ -479,17 +487,17 @@
               <UserString key="SettingName" value="tooltip delay"/>
               <UserString key="SettingValueType" value="Float"/>
             </Widget>
-            <Widget type="TextBox" skin="SandText" position="8 528 80 16">
+            <Widget type="TextBox" skin="SandText" position="8 270 80 16">
               <Property key="Caption" value="#{sFast}"/>
             </Widget>
-            <Widget type="TextBox" skin="SandText" position="370 528 80 16">
+            <Widget type="TextBox" skin="SandText" position="370 270 80 16">
               <Property key="Caption" value="#{sSlow}"/>
               <Property key="TextAlign" value="Right"/>
             </Widget>
-            <Widget type="TextBox" skin="NormalText" position="8 422 442 18" name="GuiScalingFactorLabel">
+            <Widget type="TextBox" skin="NormalText" position="8 164 442 18" name="GuiScalingFactorLabel">
               <Property key="Caption" value="#{arenamp=settings.gui_scale}"/>
             </Widget>
-            <Widget type="ScrollBar" skin="MW_HScroll" position="8 444 442 16" name="GuiScalingFactor">
+            <Widget type="ScrollBar" skin="MW_HScroll" position="8 186 442 16" name="GuiScalingFactor">
               <Property key="Range" value="151"/>
               <Property key="Page" value="1"/>
               <UserString key="SettingType" value="Slider"/>
@@ -501,10 +509,10 @@
               <UserString key="SettingLabelWidget" value="GuiScalingFactorLabel"/>
               <UserString key="SettingLabelCaption" value="#{arenamp=settings.gui_scale}"/>
             </Widget>
-            <Widget type="TextBox" skin="SandText" position="8 462 80 16">
+            <Widget type="TextBox" skin="SandText" position="8 204 80 16">
               <Property key="Caption" value="100%"/>
             </Widget>
-            <Widget type="TextBox" skin="SandText" position="370 462  80 16">
+            <Widget type="TextBox" skin="SandText" position="370 204  80 16">
               <Property key="Caption" value="250%"/>
               <Property key="TextAlign" value="Right"/>
             </Widget>
